azure-mgmt-containerservice/src/main/java/com/microsoft/azure/management/containerservice/KubernetesCluster.java

```diff
@@ -412,13 +416,27 @@
          */
         interface WithAddOnProfiles {
             /**
-             * Updates the cluster's add-on's profiles.
+             * Specifies the cluster's add-on's profiles.
              *
              * @param addOnProfileMap the cluster's add-on's profiles
-             * @return the next stage of the update
+             * @return the next stage of the definition
              */
             @Beta(SinceVersion.V1_15_0)
-            KubernetesCluster.Update withAddOnProfiles(Map<String, ManagedClusterAddonProfile> addOnProfileMap);
+            KubernetesCluster.DefinitionStages.WithCreate withAddOnProfiles(Map<String, ManagedClusterAddonProfile> addOnProfileMap);
```

Comment:
Does this change make `withAddOnProfiles` unavailable on update stage?

Reply:
There is another in update stage.